mmv1/templates/terraform/examples/blockchain_nodes_basic.tf.erb

```diff
@@ -0,0 +1,14 @@
+resource "google_blockchain_node_engine_blockchain_nodes" "<%= ctx[:primary_resource_id] %>" {
+  location = "us-central1"
+  blockchain_type = "ETHEREUM"
+  blockchain_node_id = "test"
```

Comment:
Let me introduce you to the pain of hardcoded resource names in acceptance tests :P

```
Error creating BlockchainNodes: googleapi: Error 409: Resource 'projects/ci-test-project-188019/locations/us-central1/blockchainNodes/test' already exists
```

Reply:
Here's an example (https://github.com/GoogleCloudPlatform/magic-modules/pull/9226) of where I've fixed an acceptance test that's failing because a resource has a name that is:
- hardcoded (i.e. no random bits)
- non-sweepable (i.e. doesn't start with `tf-test-`)